### ISSUE TITLE
Get-DbaServiceAccount

### ISSUE THREAD

**Issue by SirCaptainMitch** [sqlcollaborative/dbatools#584]
Is this a *feature* OR *bug*:
https://trello.com/c/qnKWqK8V

**Additional comments**
Inspired by this: https://twitter.com/n8gis/status/808651235771424768

Get Service account for Agent and SQL Server and whatever else.

**Comment by wsmelton** [comment]
This had an initial PR in #451 but was closed...so I'm not sure what the current status of this command is.

**Comment by PowerDBAKlaas** [comment]
I first got to solve Get-DbaSqlService. For this one the PR was closed by @ctrlbold but the branch is kept because one day it should work :-).
If someone could give me some details about the failures they encounter, I would be grateful.

The requirement to be a local admin remains of course.

That's why an additional Get-DbaServiceAccount could be useful too, because when we collect this via Sqlserver, users that are not local admin can get some information about the services too.
